bibliopixel/drivers/serial_driver.py

```diff
@@ -80,7 +80,11 @@ class DriverSerial(DriverBase):
     deviceIDS = {}
     deviceVers = []
 
-    def __init__(self, type, num, dev="", c_order=ChannelOrder.RGB, SPISpeed=2, gamma=None, restart_timeout=3, deviceID=None, hardwareID="1D50:60AB"):
+    def __init__(self, type, num, dev="",
+                 c_order=ChannelOrder.RGB, SPISpeed=2,
+                 gamma=None, restart_timeout=3,
+                 deviceID=None, hardwareID="1D50:60AB",
+                 baudrate=921600):
```

Comment:
What you are remembering are old RS-232 baud rates... 921600 is very standard for anything that's USB Serial. I use it all the time and it's the default for the ATMega32u4 USB stack that I use in the AllPixel.